backend.py

```diff
@@ -2,41 +2,62 @@
 from flaskext.mysql import MySQL
 from datetime import datetime
 import logic
+import json
+from collections import defaultdict
 app = Flask(__name__)
 
-app.config['MYSQL_HOST'] = 'Osamas-MacBook-Pro.local'
-app.config['MYSQL_USER'] = 'Osama'
-app.config['MYSQL_PASSWORD'] = 'CSCI5030SLU2021'
+# app.config['MYSQL_HOST'] = 'Osamas-MacBook-Pro.local'
+# app.config['MYSQL_USER'] = 'Osama'
+# app.config['MYSQL_PASSWORD'] = 'CSCI5030SLU2021'
+# app.config['MYSQL_DATABASE_DB'] = 'wordsense'
+
+app.config['MYSQL_HOST'] = 'localhost'
+app.config['MYSQL_DATABASE_USER'] = 'pnkls'
+app.config['MYSQL_DATABASE_PASSWORD'] = 'root'
 app.config['MYSQL_DATABASE_DB'] = 'wordsense'
 
 mysql = MySQL(app)
 conn = mysql.connect()
 cursor = conn.cursor()
 
+def GetIndexing():
+    f = open("indexing.txt", "r")
+    dictionary = json.loads(f.read())
+    return dictionary
+
 @app.route('/', methods =["GET", "POST"])
 def homepage():
+    # Get the indexing in memory so that we have it until the application is closed
+    dictionary = GetIndexing()
+
     if request.method == "GET":
         sentence_List = ""
         error = ""
     if request.method == "POST":
         language_selected = request.form.get('language')
         Lang_ID = logic.SQLQuery(f"SELECT Lang_ID FROM Lang_Ref WHERE Lang_Desc = '{language_selected}';")
         part_of_speech_selected = request.form.get('partOfSpeech')
-        part_of_speech_ID = logic.SQLQuery(f"SELECT Part_ID FROM Speech_Parts WHERE Lang_ID = '{Lang_ID[0]}' AND Part_Desc = '{part_of_speech_selected}';")
-        word_selected = logic.wordcreator(request.form.get('word'),request.form.get('partOfSpeech'))
-        sentence_List = logic.SQLQuery(f"select Line_Text from {language_selected}_corpus where Line_Text like  '% {word_selected} %';")
+        word_selected = (request.form.get('word') + '/' + request.form.get('partOfSpeech')).lower()
+
```

Comment:
I am not sure if GitHub is doing or if it is the way the code is written but we should get rid of the blank lines for example line 41

Reply:
I have intentionally blank lines after lines so that the code looks cleaner and easier to understand different parts. I can remove them if the team wants.

Reply:
Sounds good